domain/src/main/java/com/architect/codes/mu8/model/Comic.kt

```diff
@@ -0,0 +1,7 @@
+package com.architect.codes.mu8.model
+
+data class Comic(
```

Comment:
This data class fits in one line.

Reply:
@recaldev  When it must be in more lines?